### PR TITLE
Revert "Remove the --report-file option"

### LINKED ISSUE #176: Loading library writes to *stdout* in conflict with  '--progress none --report surefire'
When testworks loads a library writes to `*stdout*` making unusable the 'report' output.

See line
https://github.com/dylan-lang/testworks/blob/5a779d9511685293c7564de609f076da44756644/command-line.dylan#L124

### PR DESCRIPTION
**Pull request by cgay**
This reverts commit a7183e34fe82b06343cd6e7b61a796206f840f71.

Fixes #176

### CODE REVIEW

File: documentation/source/usage.rst [hunk 1]
```diff
@@ -510,20 +510,19 @@ xxx-test-suite-app.lid`` and run with ``xxx-test-suite-app --help``.
...
+report of test run results so that those results can be compared with
+subsequent test runs, for example to find regressions. These are the available
+report types:
 
-failures (the default)
-  Prints out only the list of failures and a summary, in readable text format.
-
-full
-  Like ``failures`` but prints results whether passing or failing.
+failures
```
Comment (waywardmonkeys) [inline documentation/source/usage.rst:518]: Is this change and the one below still valid and part of this restoration of `--report-file`?

**Comment by waywardmonkeys** [comment]
Should it be trying to deploy docs as part of CI for a pull request?

File: documentation/source/usage.rst (hunk 1 above)
Comment (cgay) [reply documentation/source/usage.rst:518]: I'll undo that. It was a pure revert.

**Comment by cgay** [comment]
> Should it be trying to deploy docs as part of CI for a pull request?

In general I would say no, but it deploys to GitHub Pages so it can be visually reviewed, not to opendylan.org, so I think it's ok.

File: documentation/source/usage.rst (hunk 1 above)
Comment (cgay) [reply documentation/source/usage.rst]: Done.

**Review by waywardmonkeys — APPROVED** [review]

**Pull request merged**